templates/blog/article.slang

```diff
@@ -10,17 +10,14 @@
 			.card-content
 				.media
 					.media-content.has-text-centered
-						a href="/blog/#{@title}"
+						a href="/blog/#{HTML.escape @title_markdown}"
```

Comment:
No way to do it in an action BEFORE the template? In MVC, it would be done in Controller phase.

Reply:
Can you provide code or pseudo-code to describe how this should be used?

Reply:
For an example the template use a variable. This variable comes from somewhere, probably a method that use the template to put data into.
So why not giving the template a specific variable that is already the result of HTML.escape @title_markdown?

#{escaped_title} for an example.